mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,53 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  - b is a parameter so is undeclared
   2. Remove the line that throws the error.
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  x = 2 and a = 6, d = 4 and then 6 and e = 9 and then 13
+  4. How many times is "f1" called? twice
+  5. How many times is "f2" called? three times
+  6. What value does the "a" parameter take in the first "f1" call? 7
+  7. What is the value of the "a" outer variable when "f1" is called for the first time? 7
 */
-
 let x = 2;
 let a = 6;
 
 const f1 = function (a, b) {
   return a + b;
 };
+// f1 is fine, all place holders used
 
 const f2 = function (a, b) {
   return a + b + x;
 };
 
+//expext f2 to return the sum of the place holders and x which equlas 2
 console.log(x);
 console.log(a);
-console.log(b);
+//console.log(b);
+// expect cosole.log b to retrun undefined
 
 for (let i = 0; i < 5; ++i) {
+  //++i means when evalueated the value of i is returned after it has been increamented,
```

Comment:
If these comments are notes to yourself then you can remove them when committing. But in other circumstances, it's good to leave comments for people who are going to be reading your code.